github/util.go

```diff
@@ -46,10 +46,10 @@ func validateValueFunc(values []string) schema.SchemaValidateFunc {
 }
 
 // return the pieces of id `a:b` as a, b
```

Comment:
A blurb here describing `left` and `right` may be useful to future users of this function.

Reply:
Agreed, I've pushed an update which takes care of this.